### PR TITLE
rewrite More comparisons

### PR DESCRIPTION
**Pull request by NOname-awa**
rewrite extension

### CODE REVIEW

**Comment by Skyhigh173** [comment]
Your extension id is just 'EXTENSION', you need to change it to something like 'NoNameMoreComparisons'

File: extensions/NOname-awa/More comparisons [hunk 1]
```diff
@@ -0,0 +1,750 @@
...
+                            BOOLEAN: {
+                                type: ArgumentType.BOOLEAN,
+                                defaultValue: ''
+                            }
+                        }
+                    }
+                ],
+                menus: {
+                    BOOLEAN_MENU: {
+                        acceptReporters: false,
```
Comment (LilyMakesThings) [inline extensions/NOname-awa/More comparisons:422]: Why can't these (and the following menus) be `acceptReporters: true`?

File: extensions/NOname-awa/More comparisons [hunk 2]
```diff
@@ -0,0 +1,750 @@
...
+                                }),
+                                value: 'boolean'
+                            }
+                        ]
+                    }
+                }
+            };
+        }
+        booleanMenu({ BOOLEAN }) {
+            if (BOOLEAN === 'random') return Math.random() < 0.5 ? true : false
```
Comment (LilyMakesThings) [inline extensions/NOname-awa/More comparisons:623]: You don't have to do this, but internally Scratch's "random" value is `_random_`, consider changing to that for consistency

File: extensions/NOname-awa/More comparisons [hunk 3]
```diff
@@ -0,0 +1,750 @@
...
+                return true;
+            }
+            const isComposite = (number) => {
+                if (number <= 1) return false;
+                for (let i = 2; i <= Math.sqrt(number); i++) {
+                    if (number % i === 0) return true;
+                }
+                return false;
+            }
+            switch (MODE) {
```
Comment (LilyMakesThings) [inline extensions/NOname-awa/More comparisons:650]: And you're positive this switch will always return _something_, even without a default?

File: extensions/NOname-awa/More comparisons
Comment (yuri-kiss) [inline extensions/NOname-awa/More comparisons:1]: Is this supposed to be in a new file, also please add a `.js` extension if so.

Comment (yuri-kiss) [reply extensions/NOname-awa/More comparisons:1]: also please add the default header that all the other extensions have

**Pull request closed without merging**